packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -182,7 +183,7 @@ vm_service.VmService setUpVmService(
         }
       };
     });
-    vmService.registerService('reloadSources', 'Flutter Tools');
```

Comment:
Shouldn't these have generated a lint about unawaited `Future`s?

Reply:
This method does not return a future

Reply:
(this is registering a string on this particular client, so it does not need to ping the actual vm service as far as I know)

Reply:
The PR is taking the result of `vmService.registerService('reloadSources', 'Flutter Tools')` and adding it to `registrationRequests`, which is a list of `Future`s. Then further down, the PR is doing an ` await Future.wait()` on `registrationRequests`. If I have misread this code, then my request is for more comments =)

Reply:
oh, sorry - I thought you were asking about line before that. Will add more comments :)

Reply:
Oh, the reason there was no lint surfaced is because this was not an `async` method BTW

Reply:
done